contrib/freshmaker-cli

```diff
@@ -18,14 +18,20 @@ Example usage:
         ./freshmaker-cli --redhat --stage --name httpd24-docker
     - Search for events in particular state
         ./freshmaker-cli --redhat --stage --state skipped
+    - Search for RHSA ignored by Freshmaker
+        ./freshmaker-cli --redhat  --errata
```

Comment:
Maybe it would be nice to show the example of how to pass the range dates.
Also it would be nice if it would be clearer that with this combination you get only the ignored advisories.